### PR TITLE
Return focus to the lookup button when license lookup modal is closed…

### PR DESCRIPTION
**Pull request by aditya-matukumalli**
…, refs ERM-620, ERM-622

Should be merged after https://github.com/folio-org/stripes-components/pull/1115.

Focus on the button that triggers the modal on either closing a modal or selecting a row. 

Also, changed https://github.com/folio-org/ui-plugin-find-license/pull/56/files#diff-ce4229b24108600735f6e9fe9281461cR81 to a `button` so that the rows can be tabbed through (fixes ERM-622)

### CODE REVIEW

File: src/View.js [hunk 1]
```diff
@@ -78,20 +78,19 @@ export default class View extends React.Component {
...
+      <button
         className={rowClass}
         data-label={[
           rowData.name,
           this.formatter.type(rowData),
           this.formatter.status(rowData),
         ].join('...')}
         key={`row-${rowIndex}`}
-        role="row"
+        type="button"
```
Comment (doytch) [inline src/View.js]: What's the purpose of these changes? They significantly decrease a11y.

Comment (aditya-matukumalli) [reply src/View.js]: The app results need to be rendering out something thats tab-able to fix https://issues.folio.org/browse/ERM-622. Hence changed this to a button. Are you specifically referring `role="row`? If so, thats coming through already via the `rowProps`

Comment (doytch) [reply src/View.js]: Ahh OK. Does `aria-rowindex` get passed in as well?

Comment (aditya-matukumalli) [reply src/View.js]: Yes, `aria-rowindex` gets passed as well
